spec/datadog/appsec/reactive/operation_spec.rb

```diff
@@ -9,22 +9,22 @@
     Thread.current[:datadog_security_active_operation] = nil
```

Comment:
I would suggest hiding the implementation detail behind a private class-level `reset!` method. This way the implementation detail is self-contained in the class's implementation and the spec is oblivious to it.

Here's an example: https://github.com/DataDog/dd-trace-rb/blob/20616adb41018c0ab947ce3327845652f5db9da5/lib/datadog/appsec/configuration/settings.rb#L204-L207